spec/injjject_spec.rb

```diff
@@ -0,0 +1,20 @@
+
+require 'injjject.rb'
+
+describe Array do
+  subject { [1, 2, 3, 4] }
+
```

Comment:
You're challenge won't work with a symbol
` [1,2,3].inject(:+)` It would be nice to see this.